packages/kbn-ambient-ui-types/types/index.d.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export * from './file_system_cache';
```

Comment:
This feels very storybook specific and not appropriate for this package.

Reply:
I also preferred the single file version of this module. All these modules are tiny and it makes comprehending this package much more difficult IMO.

Reply:
Sure thing.  I originally had them split in order to add tests to each one, but given the need for an additional dependency (`expect-type`), I'll defer.